sdr/index.md

```diff
@@ -3,26 +3,130 @@
+
+### Workflow Server
+_a Rails App to track the workflows associated with an object, and the state of each step and an error string for failed steps_
+
+(ask JCoyne)
+
+* workflow-server-rails - _the Rails app_
+* dor-workflow-client - _Ruby gem for making calls to workflow app_
 
 ### SDR-API
+
 This is an external facing API. It is not totally complete as it doesn't produce derivatives for book/image types and it doesn't yet have fined grained authorization.  Currently all of the h2 and GoogleBooks deposits flow through this API.  This in turn communicates with dor-services-app on the back end
 
+(ask JCoyne)
+
+* sdr-client - _Ruby gem for making calls to sdr-api app_
+
 ### ETD
 Students can deposit thesis and dissertations into the repository
 
 ### h2
-Stanford affiliated users can self deposit. The previous iteration of this software was called "Hydrus", the newest version is "h2".
+Stanford affiliated users can self deposit. The previous iteration of this software was called "Hydrus", the newest version is "h2" (nicknamed Happy Heron).
 
-### dor-indexing-app
-Creates a solr representation of the objects and writes it to Solr. This solr index is used for Argo and for dor-services-app since the Fedora 3 repository has no query capabilities.
+(ask Mike or JCoyne)
+
+infrastructure-integration-test's `create_object_h2_spec.rb` can provide a walkthrough for how to do a self-deposit
+
+### Google Books
+
+Automatically downloads books we have sent off to google for scanning.  This provides a minimal UI for monitoring and is only used by the infrastructure team and Andrew.  Google books are deposited as a single ZIP file with all the images.  They are kept dark, as many of them are not out of copyright. The downloading job uses the SDR-API to deposit the files.
 
 ### Pre-assembly
+
 ### preservation
-### robots
+
+#### preservation_catalog
+
+#### preservation_robots
+
+see robots section below
+
+### Robot Suites
+
+Different specialized robot codebases perform the individual steps of accessioning, preservation, making available for access, etc.  More about how they hand off to each other and interact with other services [here](concepts_and_interactions#Workflows-and-Robots)
+
+#### robot-console
+_Tiny app providing the resque console to view the state of our robot redis queues, activity and errors_
+  * e.g. https://robot-console-prod.stanford.edu/overview
+  * https://github.com/sul-dlss/robot-console/
+
+#### common-accessioning
+_Suite of robots that handle the tasks of accessioning digital objects_
+
+(ask Peter)
+
+Workflows implemented include `assemblyWF`, `accessionWF`, `goobiWF`, `disseminationWF`, `releaseWF`
+
+* https://github.com/sul-dlss/common-accessioning/
+
+#### preservation_robots
+_robots to preserve objects being accessioned via `preservationIngestWF` formerly known as `sdrIngestWF`_
```

Comment:
I think it's important to add here that these robots run on a VM with access to the filesystems used by preservation. This is the main reason they are separated from common-accessioning.